Patches/JPT Open The Windows/Openvindows.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<Patch>
+	<Operation Class="PatchOperationFindMod">
+		<mods>
+		<li>[JPT] Open The Windows</li>
+		</mods>
+		
+		<match Class="PatchOperationSequence">
+			<success></success>

```

Comment:
Remove this line, please--it's unnecessary and can only stand to cause weird behavior.